feat: added 'add review' comment

wannacfuture opened the pull request:
# Description

- I added 'Please add review' comment after payment permit generation.

closes #307

pavlovcik commented:
> I added 'Please add review' comment before payment permit generation.

What was your reasoning for putting it before? I do not have a strong opinion on this matter, but I feel that I would put it at the very end to make sure that the comment is not missed.

wannacfuture commented:
> > I added 'Please add review' comment before payment permit generation.
> 
> What was your reasoning for putting it before? I do not have a strong opinion on this matter, but I feel that I would put it at the very end to make sure that the comment is not missed.

Yeah, seems u're right. Fixed.

pavlovcik commented on src/handlers/comment/handlers/index.ts:
```diff
@@ -42,7 +42,11 @@ export const issueClosedCallback = async (): Promise<void> => {
   try {
     const comment = await handleIssueClosed();
-    return await addCommentToIssue(comment!, issue!.number);
+    await addCommentToIssue(comment!, issue!.number);
+    await addCommentToIssue(
```
It seems that these should be consolidated into a single comment. What is :45 commenting exactly?

wannacfuture replied:
:45 is the returning comment about payment permit when the user closed issue. And if we consolidate them into one, it will be hard to impact review comment. Is that OK?
I mean, the payment permit result and "If you enjoy the experience ..." comment will be one comment.

pavlovcik replied:
Do you have an example on your fork so I could see and compare?
